src/test/ui/issue-49579.rs

```diff
@@ -0,0 +1,27 @@
+// Copyright 2013 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+
+// compile-pass
+
+#![feature(nll)]
```

Comment:
Does this test *need* to utilize `#![feature(nll)]`? Or can we leave that of, and let the `compare-mode=nll` handle testing  that case (and thus get a test for this in both AST-borrowck and NLL modes)

Reply:
This test does not pass without `#![feature(nll)]`. I'm not sure if it is possible to allow a UI test to compile successfully in one compare mode but fail in another.

Reply:
You can use `#[rustc_error]` to support that. There are examples of it elsewhere in the `ui/` test suite.

Reply:
but having said that, there's no reason to worry about it in this case. If the test does not compile w/o `#[feature(nll)]`, that is fine.